src/middleware/errors.json

```diff
@@ -118,22 +108,34 @@
     "userMessages": "",
     "statusCode": 400
   },
+
+
   {
-    "name": "optionsValidationError",
-    "code": "17",
-    "developerMessage": "One of the supplied options is invalid",
-    "statusCode": 400
+    "name": "SequelizeConnectionError",
```

Comment:
Can this one not be removed now that we use cosmos?

Reply:
Yes it's not used anymore so I will remove